core_tic_tac_toe/display.py

```diff
@@ -39,9 +39,18 @@ def announce_winner(self, player_name):
     def announce_tie(self):
         self.io.print_message("Good game, but it's a tie!")
 
+    def welcome_message(self):
+        self.io.print_message("Welcome to Tic Tac Toe\n")
```

Comment:
Noticing that this one has a `\n`, but the others don't. I don't think this class should know what is coming next, and the classes using these methods shouldn't have to know which ones have a `\n`. I wonder if having a `emptyLine()` could help.

Reply:
Looking at it more closely, I don't think the \n is even necessary, but I'd like to discuss emptyLine()